src/ansi.rs

```diff
@@ -742,13 +765,22 @@ impl<'a, H, W> vte::Perform for Performer<'a, H, W>
         _ignore: bool,
         byte: u8
     ) {
+        let i1: u8 = match intermediates.first() {
+            Some(i) => *i,
+            None => 0,
+        };
+
         match byte {
+            b'B' if i1 == b'(' =>
```

Comment:
I was a bit worried about how the compiler would treat the match statement after this. It looks like we still get a jump table for each variant, and the cases with `if expr` just have a comparison added in. Nifty.